File: code_llm/multiagentes/gpt-4o-mini/Relógio.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    int H, M, S, T;
    cin >> H >> M >> S >> T;

    int total_seconds = H * 3600 + M * 60 + S;
    int new_total_seconds = total_seconds + T;

    int new_H = (new_total_seconds / 3600) % 24;
    int new_M = (new_total_seconds / 60) % 60;
    int new_S = new_total_seconds % 60;

    cout << new_H << endl;
    cout << new_M << endl;
    cout << new_S << endl;

    return 0;
}

File: output/gemini_3_1_pro_preview/few/python/Relógio.py
import sys

input_data = sys.stdin.read().split()
if not input_data:
    exit()

h = int(input_data[0])
m = int(input_data[1])
s = int(input_data[2])
t = int(input_data[3])

total_seconds = h * 3600 + m * 60 + s + t

new_s = total_seconds % 60
total_seconds //= 60
new_m = total_seconds % 60
total_seconds //= 60
new_h = total_seconds % 24

print(new_h)
print(new_m)
print(new_s)